components/interview/icons/LucideAddNode.tsx

```diff
@@ -0,0 +1,42 @@
+import { cn } from '~/lib/utils';
+
+import React, { lazy, Suspense } from 'react';
+import type { LucideProps } from 'lucide-react';
+import dynamicIconImports from 'lucide-react/dynamicIconImports';
+import { Circle } from 'lucide-react';
+
+// do we need a fallback for this?
```

Comment:
No, the way this should be done is using `dynamic()` from `next/dynamic`. So:

```
const LucideIcon = dynamic(dynamicIconImports[name])
```

This should mean that the icon is rendered in the SSR phase, and so we won't see a flash that would need either a suspense boundary or custom loading component.